### DISCUSSION TITLE
How to parse this data on Grafana

### DISCUSSION THREAD

**Discussion by gagiuntoli** [grafana/grafana-infinity-datasource#510] [Q&A]
I have the result of the following GraphQL query on the Query Inspector -> Data:

```json
{"trades":[{"timestamp":"1668617006","tradeAmount":"100000000"},{"timestamp":"1673707542","tradeAmount":"100000000"}]}
```

I want to parse it an create a table:

```
timestamp | tradeAmount
1668617006 | 100000000
...
```
I am stuck at this point using the backend parser. This is my query:

```
{
  trades (first: 50) {
  	timestamp
	  tradeAmount
	}
}
```

And my endpoint: `https://api.thegraph.com/subgraphs/name/sportsfi-ubet/ubetmumbai-sandbox`

![image](https://user-images.githubusercontent.com/25662036/213416577-45a1a9e0-1637-4ba4-a8b6-42135f954022.png)

**Comment by aakashH242** [comment]
You can use the UQL parser to extract the fields and create a table out of them.
Try using - 
```
json-parser
| scope "trades"
```
Hope it helps you!

![image](https://github.com/grafana/grafana-infinity-datasource/assets/20237354/a25b64d9-fbc1-4f23-8a05-774785edceda)
